### PR TITLE
Fix multiple expressions bug, add test

### PR DESCRIPTION
**Pull request by drwpow**

### CODE REVIEW

File: internal/token_test.go [hunk 1]
```diff
@@ -270,6 +270,11 @@ func TestExpressions(t *testing.T) {
 			"{`${`${`${foo}`}`}`}",
 			[]TokenType{StartExpressionToken, TextToken, TextToken, TextToken, TextToken, TextToken, EndExpressionToken},
 		},
+		{
+			"element with multiple expressions",
+			"<div>Hello {first} {last}</div>",
```
Comment (drwpow) [inline internal/token_test.go:275]: This was failing before. Basically, when entering expression context, we would enter “text” mode but fail to exit.

File: internal/token.go [hunk 2]
```diff
@@ -1602,6 +1602,7 @@ expression_loop:
 				z.tt = TextToken
 				return z.tt
 			}
+			z.openBraceIsExpressionStart = true
```
Comment (drwpow) [inline internal/token.go:1605]: Not 100% sure this is in the right place but all tests seem to be passing 🤷🏻. And we can move with other tests.

Comment (natemoo-re) [reply internal/token.go:1605]: Noice! LGTM.

**Review by natemoo-re — APPROVED** [review]
LGTM!

**Pull request merged**